crates/astria-bridge-withdrawer/src/withdrawer/ethereum/watcher.rs

```diff
@@ -450,19 +520,26 @@ mod tests {
         };
         let denom: Denom = Denom::from_base_denom("nria");
         let expected_action =
-            event_to_action(expected_event, denom.id(), denom.clone(), 1).unwrap();
+            event_to_action(expected_event, denom.id(), denom.clone(), 10u128.pow(18)).unwrap();
```

Comment:
```suggestion
            event_to_action(expected_event, denom.id(), denom.clone(), 1).unwrap();
```

after the logic for the divisor above is fixed